src/homepageExperience/containers/HomepageContainer.tsx

```diff
@@ -180,21 +182,23 @@ export const HomepageContainer: FC = () => {
                         </div>
                       </Link>
                     </ResourceCard>
-                    <ResourceCard style={cardStyle}>
-                      <Link
-                        to={arduinoLink}
-                        style={linkStyle}
-                        onClick={logArduinoWizardClick}
-                      >
-                        <div
-                          className="homepage-wizard-language-tile"
-                          data-testid="homepage-wizard-language-tile--arduino"
+                    {!isFlagEnabled('ioxOnboarding') && (
+                      <ResourceCard style={cardStyle}>
```

Comment:
Why are we only showing the arduino onboarding in iox?

Reply:
Not showing Arduino in IOx because the library doesn't support it yet.